docs/readmes/dp/debug_logs.md

```diff
@@ -26,6 +189,23 @@ domain-proxy-radio-controller-5c868696d9-s7vgg          1/1     Running     0
 
 ## Check individual pods' logs
 
+To view logs from individual Domain Proxy pods, execute `kubectl logs` command with one of the pod
+names listed in [previous chapter](#listing-domain-proxy-pods-in-kubernetes)
+
+* `Radio Controller` (RC) logs:
+  * logs related to AGW/enodebd <-> Domain Proxy communication
+  * logs related to requests generated by Active Mode Controller (Domain Proxy logic, which generates appropriate SAS requests)
+  * logs related with Database modifications, which where the result of incoming API calls (either from AGW or AMC)
+* `Configuration Controller` (CC) logs:
+  * logs related to Domain Proxy <-> SAS communication
+  * logs related with Database modifications, which were the result of processing SAS responses
+* `Active Mode Controller` (AMC) logs:
+  * logs related to internal business logic of Domain Proxy functionality
+  * logs related to Database state management
+  * logs related to generated SAS requests based on Database state
```

Comment:
[Hyphens should be preferred over asterisks](https://github.com/magma/magma/wiki/Contributing-Code-Conventions#markdown).